docker/install/install_tvm_gpu.sh

```diff
@@ -29,6 +29,7 @@ git checkout 4b13bf668edc7099b38d463e5db94ebc96c80470
 echo set\(USE_LLVM llvm-config-8\) >> config.cmake
 echo set\(USE_CUDA ON\) >> config.cmake
 echo set\(USE_CUDNN ON\) >> config.cmake
+echo set\(USE_CUSPARSE ON\) >> config.cmake
```

Comment:
I don't think CI uses this file to install TVM tho. You probably need to ask an admin to do so.

Reply:
Oh.. I see. Thanks!

Reply:
Yes, we'll need to ask @tqchen to re-generate the docker image for the CIs

Reply:
The checking also stuck on the the point needing the cusparse modules.
@tqchen Could you re-generate the CIs with USE_CUSPARSE ON in the config.cmake?